docs/dev-concepts/participant-authentication.md

```diff
@@ -0,0 +1,76 @@
+
+<!--In the first sentence below, is it the XMTP client SDK that enables this - or the client/client protocol? I think the client provides this functionality based on the client protocol and the client SDK exposes it? What might make most sense to a developer reader here? I ask because I am seeing the opportunity, or need, the make a clearer division in the docs between the protocol and client -- and the SDK. For example, many of these topics in the Development concepts section are specifically about the protocols and could be refined to make this clearer.-->
+
+The XMTP client SDK enables client apps to establish secure, unfalsifiable relationships between their users' blockchain accounts such that only the sender and recipient can decrypt messages sent between them.
+
+These relationships are established by authenticating a user's blockchain account using the keys described here.
+
+## Identity keys
+
+A message API client (client) and the XMTP network cannot (and should not) access a user's blockchain account keys. For this reason, a client generates the following identity keys to serve as proxies for a user's blockchain account keys:
+
+- `IdentityPublicKey` (also known as an XMTP identity)
+- `IdentityPrivateKey`
+
+A client prompts the user to sign the `IdentityPublicKey` with their blockchain account keys to confirm, or prove, that the user, as controller of the blockchain account, generated the XMTP identity.
+
+Anyone can verify the user's signature and consider it as an attestation that the `IdentityPublicKey` is the user's true proxy keys since only the user's blockchain account keys could have created the signature.
+
+## Prekeys
+
+A client also generates public and private prekeys for use with a user's `IdentityPublicKey`, or XMTP identity.
+
+A client prompts the user to sign their public prekey with their `IdentityPrivateKey`.
+
+<!--Below, looking for an accurate positioning for prekeys - as signing on behalf of an XMTP identity? Something like - the prekeys belong to, or as associated with, the XMTP identity? Is the idea that the prekeys - once signed by the identity keys - are the keys that are doing all the work? And that's why being able to rotate them in the future is important? Whereas, the identity keys sign the prekeys, but then aren't actively being used to encrypt and sign anything. And the blockchain account keys after signing the identity keys are REALLY out of the picture. I am seeing something like layers of defense - or obfuscation - or abstraction - is this how it is meant?-->
+
+The prekeys are used to encrypt invitations and sign encoded messages on behalf of the XMTP identity.
+<!--I don't explicitly mention the prekey use in the invitation encryption flows. I mention the public and private key bundles - when we talk about these bundles - are we specifically talking about the public and private prekeys? https://xmtp.org/docs/dev-concepts/invitation-and-message-encryption#invitation-encryption.-->
```

Comment:
Not quite sure if I get the question, but both public and private keys play a role in X3DH.